scripts/ci/run-monitor-ci-tests.bash

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
```

Comment:
This is a direct copy of the script in `ui`: https://github.com/influxdata/ui/blob/master/scripts/run-monitor-ci-tests.bash

Reply:
I like having a copy here because:
1. We don't have to worry about our tests tipping over if some sort of breaking change is made to the script in `monitor-ci`
2. If we feel motivated, we can simplify the script to only contain the OSS-relevant pieces, making it easier to understand what's happening

That said, if we expect other teams will be making a lot of non-breaking improvements to the script in `monitor-ci` then I could be convinced pulling the latest each time is the right move. In that case a shallow git clone sounds like a good path :+1:

Reply:
Good points Dan...theoretically it should only depend on the CircleCI API not changing, so it's not like there is any kind of other special information needed from the `ui` repository to make it work. I don't think we'll need to update it very often, and having our own copy might allow us to avoid having to change *three* repos if we want to update how OSS interacts with `monitor-ci`.

Since I've already copied the script here, I think I'll leave it here for now. If I find it too annoying to have to keep updating all the time I'll revisit it later.